ansible/roles/omero-python-deps/tasks/pypi.yml

```diff
@@ -0,0 +1,20 @@
+---
+# Install python modules from pypi
+
+- name: omero python deps | pip compilation requirements
+  become: yes
+  yum:
+    name: "{{ item }}"
+    state: present
+  with_items:
+  - libjpeg-devel
+  - libtiff-devel
+  - zlib-devel
+
+- name: omero python deps | pypi pillow
+  become: yes
+  pip:
+    name: "{{ item }}"
+    state: present
+  with_items:
+  - "Pillow==2.9"
```

Comment:
could this go to `{{ omero_py_pip_dependencies }}` with an option `when: omero_py_pip_dependencies`